corehq/apps/app_manager/templates/app_manager/partials/releases/releases_deploy_modal.html

```diff
@@ -167,7 +167,7 @@ <h4 class="panel-title">
                                 Note: Offline install automatically installs all multimedia
                                 included in your form. All subsequent remote updates will also
                                 include multimedia which could require a lot of data. See the
-                                <a href='https://confluence.dimagi.com/display/commcarepublic/Install+CommCare+for+Android+Smartphones'>
+                                <a href='https://dimagi.atlassian.net/wiki/spaces/commcarepublic/pages/2143946239/Installing+CommCare+on+Android+Devices'>
```

Comment:
nit: double quotes in html

```suggestion
                                <a href="https://dimagi.atlassian.net/wiki/spaces/commcarepublic/pages/2143946239/Installing+CommCare+on+Android+Devices">
```